compiler/src/iree/compiler/Codegen/Common/test/iree_comprehensive_bufferize.mlir

```diff
@@ -79,7 +79,8 @@ func.func @matmul_fill() {
   %m = hal.interface.constant.load[0] : index
   %n = hal.interface.constant.load[1] : index
   %k = hal.interface.constant.load[2] : index
-  %base_offset = hal.interface.constant.load[3] alignment(8) : index
+  %base_offset_i32 = hal.interface.constant.load[3] alignment(8) : i32
+  %base_offset = arith.index_castui %base_offset_i32 {stream.alignment = 8 : index} : i32 to index
```

Comment:
The attribute could be dropped though.... do we need to use this, or can we just walk the use-def chain to the load and get the aligment?

Reply:
ah good catch - had fixed this locally but not pushed - done!